cohortextractor/patients.py

```diff
@@ -243,17 +243,33 @@ def most_recent_bmi(
     weight measurement for this.
 
     Args:
-        on_or_before (str, None): String of Data in YYYY-MM-DD format or None,
-        on_or_after (str, None): String of Data in YYYY-MM-DD format or None,
-        between (list, None): List of two date strings as YYYY-MM-DD format or None,
-        minimum_age_at_measurement (int): Minimum age at measurement. Default value is 16.
-        return_expectations (dict, None): An expectation definition defining an incidence
-            and a distribution as a dict,
-        include_measurement_date (bool): Flag to return an additional column of date of bmi. Set as default to False
-        date_format (str): Date String of format to return date of measurement if flagged
-        include_month (bool): Flag for how granular to return date
-        include_day (bool): Flag for how granular to return date
-
+        on_or_before: Date of interest as a string with the format "YYYY-MM-DD". This will find data
```

Comment:
What does "will find" mean? Can we be more specific? How about "restricts filter to consider data on or before the given date"?

Reply:
I am not sure "restricts filter to consider data on or before the given date" is that clear. What about "This  filters to data on or before the given date"

Reply:
I see what you mean. "filters results to measurements on or before the given date"?

(I still have a preference for losing the word "this")